packages/vuepress-theme-nimatron/styles/index.scss

```diff
@@ -257,3 +257,39 @@ h1 {
     margin: 0;
   }
 }
+
+.content__default { // stylelint-disable
+  > h1,
+  > h2,
+  > h3,
+  > h4,
+  > h5,
+  > h6 {
+    position: relative;
+
+    .header-anchor {
+      position: absolute;
+      left: -30px;
```

Comment:
Where's this `30px` come from?

Reply:
Totally made it up. This basically shows the anchor tag link when a headline is hovered over (see screenshot)

Will probably remove this all together.

![image](https://user-images.githubusercontent.com/62259644/80547435-082f9c80-896d-11ea-9cea-36a661dc108e.png)